src/content/docs/infrastructure/amazon-integrations/manage-aws-data/aws-integrations-metrics.mdx

```diff
@@ -11,68 +11,89 @@ redirects:
 freshnessValidatedDate: never
 ---
 
-We recommend using the [AWS CloudWatch Metric Streams integration](/docs/integrations/amazon-integrations/aws-integrations-list/aws-metric-stream/) to ingest AWS services' metrics. For a complete list of available metrics, check the CloudWatch metrics listing documentation for each AWS service.
+Once you're collecting AWS data, you can set up alert conditions or query your data in the New Relic platform. This doc describes how we store AWS dimensional metrics so you can locate AWS namespace entities and create alerts and queries about the data that matters to you. This doc then provides you with example queries and alerts to get you started monitoring your AWS environment. 
 
-## Dimensional metric naming convention [#metric-naming-convention]
+If you haven't yet, we recommend installing or migrating to the [AWS CloudWatch Metric Streams integration](/docs/integrations/amazon-integrations/aws-integrations-list/aws-metric-stream/) to ingest AWS services' metrics. If you're using API polling to collect your AWS data, you can review the metrics we collect for each integration in our [API polling metrics](#aws-metrics-table) section. 
 
-Metrics received from AWS CloudWatch are stored in New Relic as [dimensional metrics](/docs/telemetry-data-platform/ingest-manage-data/understand-data/new-relic-data-types/#dimensional-metrics) following this convention:
+## Queries, metric storage, and mapping [#query-experience]
 
-* Metrics are prefixed by the AWS namespace, all lowercase, where `/` is replaced with `.`:
-  * `AWS/EC2 -> aws.ec2`
-  * `AWS/ApplicationELB -> aws.applicationelb`
-* The original AWS metric name with its original case:
-  * `aws.ec2.CPUUtilization`
+We've mapped metrics from individual cloud integrations to new mappings from AWS CloudWatch Metric Streams. You can use either metric naming conventions and your alerts, dashboards, and queries will continue to work. Keep in mind that 
```

Comment:
"You can use either metric naming conventions" should probably be "naming convention"?

Reply:
Should be a : after "Keep in mind that"